liquidator/package.json

```diff
@@ -1,10 +1,29 @@
 {
-  "name": "liquidator",
-  "version": "0.1.0",
-  "description": "Liquidator",
+  "name": "@umaprotocol/liquidator",
+  "version": "1.0.0",
+  "description": "UMA Liquidator",
   "private": true,
   "dependencies": {
-    "winston": "^3.2.1",
-    "winston-transport": "^4.3.0"
+    "dotenv": "^6.2.0"
+  },
+  "devDependencies": {
+    "sinon": "^9.0.2",
+    "winston": "^3.2.1"
+  },
+  "homepage": "https://umaproject.org",
+  "license": "AGPL-3.0-or-later",
+  "main": "index.js",
```

Comment:
Question: Should we make it a point to fill out `main` property for each `package.json` ?

Reply:
I think so. they all have a specific entry point

Reply:
Cool

Reply:
From what I understand `main` is what makes `require` work the way you expect: https://docs.npmjs.com/files/package.json#main